administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -22,7 +22,7 @@ function iframeRef( frameRef ) {
 
 	function triggerSave() {
 		var inside = iframeRef( document.getElementById('target-association') );
-		inside.getElementById('applyBtn').click();
+		window.frames[1].Joomla.submitbutton('article.apply');
```

Comment:
The save action is working but it doesn't load the layout to modal again, my previous implementation with the "inside.getElementById('applyBtn').click();" worked only for contacts, my questions are:
- Can my current implementation be tweaked?
- Should other core components implement my older implementation?
- Are both of these solutions wrong?